### ISSUE TITLE
Table filters do not update the user on state change

### ISSUE THREAD

**Issue by kraay** [telerik/kendo-ui-core#4861]
### Bug report

On updating any filter state, no screen reader input is provided to the user. 

### Reproduction of the problem
Mock application: https://runner.telerik.io/fullscreen/oXokIKeS
Code: https://dojo.telerik.com/oXokIKeS
Steps:
Navigate to Application
Enable different filter options
Observe screen reader feedback

### Current behavior
Although current state is supplied upon landing on filter, no screen reader feedback is provided up changing the filter state.

### Expected/desired behavior
Changes to filter state should be communicated via screen reader, such as - 'Ascending' 'Descending' and 'Selected'

### Environment

* **Kendo UI version:** 2015.2.805
* **Browser:** Chrome 70.0.3538.77
* **Assisstive Technology:** NVDA

**Comment by matanasov** [comment]
The reported Kendo version (2015.2.805) is quite old. Many accessibility improvements were implemented in later versions and I would suggest that you re-verify the problem with latest version (2019.1.220).
Here is a modified DoJo:
https://dojo.telerik.com/@martinpz/oTapEkAP

**Comment by kraay** [comment]
@matanasov I changed the Kendo version in the provided sample and ran NVDA. When I click on the column names, they are sorted in ascending or descending order but this is not communicated via NVDA. Please check and reopen the bug.

**Comment by MilenaCh** [comment]
We are aware about an issue with announcing of sort attributes in IE and Edge and have already logged a [bug ](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/20581320/)in Microsoft platform about that.  
In addition, I've notice that screen readers announce properly sort attributes when navigate to a sorted header cell using keyboard in the rest of the browsers(except IE and Edge), while the same attributes are not announced if focus them with a mouse.  In general, the main use case of the screen readers is to announce the content while the user navigates with keyboard, so most probably this is the reason about such an inconsistent behavior, however more detailed information about that could be obtained from the corresponding screen reader support team.
